frontend/pro-kabaddi-league-trophy/style.css

```
@@ -0,0 +1,887 @@
+/* ==========================================================================
+   INCREDIBLE INDIA EXPLORER — PRO KABADDI LEAGUE TROPHY STYLESHEET
+   Issue: #2549
+   ========================================================================== */
+
+@import url("../../styles.css");
```

Stylelint (17.14.0)

[error] Expected "url("../../styles.css")" to be ""../../styles.css"" (import-notation)

(import-notation)